notebooks/py_scripts/00_Setup.py

```diff
@@ -426,30 +426,48 @@
 #
 
 # +
-import os
 import datajoint as dj
 from spyglass.settings import SpyglassConfig
 
-# change to the root directory of the project
-if os.path.basename(os.getcwd()) == "notebooks":
```

Comment:
By skipping the chdir, the local would no longer be saved in the spyglass root dir, and instead in the notebooks dir

I previously set the root as the place to save the local bc it's easy to copy paste the example file, but that's arguably a branching path people don't actually take, when we want to be silo-ing